src/Tasks/ResolveProjectBase.cs

```diff
@@ -293,6 +293,7 @@ protected void AddSyntheticProjectReferences(string currentProjectAbsolutePath)
                     item.SetMetadata("ReferenceOutputAssembly", "false");
                     item.SetMetadata("LinkLibraryDependencies", "false");
                     item.SetMetadata("CopyLocal", "false");
+                    item.SetMetadata("SkipGetTargetFrameworkProperties", "true");
```

Comment:
Is this necessary if ReferenceOutputAssembly=false now implies SkipGetTargetFrameworkProperties?

Reply:
Not strictly, but I can't think of a compat problem with doing this, while I can maybe conceive of one with the other approach, so I decided to do both and possibly back out the other.

Reply:
I would keep the other, too. I can't think of a scenario where you want to pick the best TFM but not reference the output assembly. SkipGetTargetFrameworkProperties was intended as an escape hatch rather than something you needed to spell out in real scenarios.